include/wx/textctrl.h

```diff
@@ -103,7 +103,12 @@ const wxTextCoord wxInvalidTextCoord    = -2;
 // wxTextCtrl file types
 // ----------------------------------------------------------------------------
 
-#define wxTEXT_TYPE_ANY     0
+enum wxTextCtrlFileType
```

Comment:
I think we ought to use scoped enum for this one.

Reply:
I was wanting to do that originally, but that would break the API for `LoadFile(const wxString&, int)` and `SaveFile(const wxString&, int)`; the `int` would need to become a `wxTextCtrlFileType`. I could do some static_cast'ing inside these functions, but that defeats the purpose of using a scoped enum. Thoughts?

Reply:
It's difficult to know if anybody really passes an `int` to these functions, knowing that there was just a single possible value for it so far, but I guess you're right and somebody must be doing it or, worse, passing just `0` here, so I take this back, let's keep your original approach, sorry.